app/views/flood_risk_engine/enrollments/steps/_add_exemptions.html.erb

```diff
@@ -1,6 +1,7 @@
 <p class="panel panel-border-wide">
   <%= t("flood_risk_engine.enrollments.steps.add_exemptions.lead_paragraph.text_1") %>
-  <br></br>
+  <br>
+  </br>
```

Comment:
[`<br>` is a self closing tag.](http://stackoverflow.com/questions/4288460/html-whats-the-correct-form-of-br) So it should be either `<br>` or `<br><br>' (which is two breakes).